test/type/_fonts.scss

```diff
@@ -80,6 +80,60 @@ config.$font-path: data.$font-path;
     }
   }
 
+  // Compile Fonts [function]
+  // ---
+   @include describe('Compile Fonts [function]') {
+    @include it('References global fonts') {
+      @include assert-equal(
+        fonts.compile-fonts((a: '#UniFont')), 
+        (a: fonts.font('UniFont'))
+      );
+    }
+
+    @include it('Compiles a local map with a global reference') {
+      @include assert-equal(
+        fonts.compile-fonts(('new': '#TestFont')), 
+        (
+          'new': (
+            'stack': (
+              TestFont,
+              Georgia,
+              serif
+            ),
+            'name': TestFont
+          )
+        )
+      );
+    }
+
+      @include it('Compiles a local map with a local reference') {
+        @include assert-equal(
+          fonts.compile-fonts(('new': '#TestFont'), 
+          (
```

Comment:
Seems like there are some indentation issues here. I guess we don't have a linter handling that…

Reply:
cleaned up this formatting